tests/integration/conftest.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from poglink.bot import ConfigurableBot
+
+
+@pytest.fixture()
+def sample_bot(bans_url_1, rates_url_1):
+    bot = ConfigurableBot(
+        ".",
+        {
+            "bans_url": bans_url_1,
+            "rates_urls": [rates_url_1],
```

Comment:
This previously wasn't a list-type, it was just the single string. 

Come to think of it, maybe it'd be worthwhile to add some logic that makes sure if `rates_urls` is passed as a stirng type, it is converted to a list with a single string element (and a test to ensure this is the case too). We have lots of outer layers making sure it's a list (like the sample config, default values, and argparse setup), but at a low-level there's nothing stopping someone from accidentally providing a string as this argument. The problem here is that it's not likely to raise an error as you can still iterate over a string; it just goes over every character.

Reply:
Agreed, we should definitely resolve that.

Reply:
#42